### DISCUSSION TITLE
Parallel processing of manifest?

### DISCUSSION THREAD

**Discussion by stuvet** [ropensci/targets#890] [Ideas]
Is there any opportunity to speed up the pipeline by offering parallel processing of the manifest? (not sure if that's what I mean - the process submitting jobs to the workers). It seems to be the limiting factor on some systems.

Thanks.

**Comment by wlandau** [comment]
This may or may not be a problem parallel computing can solve. Various package functions tend to slow down if you have several thousand targets or a slow file system. If you can supply an example where `proffer::pprof(targets::tar_manifest(callr_function = NULL))` is slow and then post the flame graph, there may be something we can do.

**Reply by stuvet** [reply]
Thanks, I'll take a look.
